interface/new/new_comprehensive_save.php

```diff
@@ -28,11 +28,8 @@
 // other interfaces can still read the data during this lock, however
 // sqlStatement("lock tables patient_data read");
 
-$result = sqlQuery("SELECT MAX(pid)+1 AS pid FROM patient_data");
-
-$newpid = 1;
-
-if ($result['pid'] > 1) $newpid = $result['pid'];
+$result = sqlQuery("SELECT id AS pid FROM patient_data ORDER BY id DESC LIMIT 1");
+$newpid = ($result['pid'] > 1) ? $result['pid'] + 1 : 1;
```

Comment:
This doesn't look like what you have on your prior PR:
https://github.com/openemr/openemr/pull/734